spec/web_helpers.rb

```diff
@@ -0,0 +1,31 @@
+module User_helpers
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)